packages/server/src/api/controllers/row/views.ts

```diff
@@ -29,49 +32,29 @@ export async function searchView(ctx: UserCtx<void, SearchResponse>) {
     undefined
 
   ctx.status = 200
-  const result = await quotas.addQuery(
-    () =>
-      sdk.rows.search({
-        tableId: view.tableId,
-        query: view.query || {},
-        fields: viewFields,
-        ...getSortOptions(ctx, view),
-      }),
-    {
-      datasourceId: view.tableId,
-    }
-  )
+
+  const searchOptions: RequiredKeys<SearchViewRowRequest> &
```

Comment:
Doing so will make sure that the request object and the SDK parameters match, and adding new fields in the request body will require a specific mapping in here